Himanshu_Sharma/project1-tic_tac_toe/assets/scripts/script.js

```diff
@@ -0,0 +1,229 @@
+//This obj stores the state of any block 
+var blocks = {};
+
+//A simple boolean to check turn
+var turn = false;
+
+(function blocksCreation(){
+    for(let row = 0; row < 3; row++){
```

Comment:
time complexity is 'n^2', can you improve it to 'n'

Reply:
It cannot possibly, for current built project as it is necessary to add a unique class to each block of the game! I will take care of it later...